docs/source/torch.rst

```diff
@@ -133,6 +133,7 @@ Indexing, Slicing, Joining, Mutating Ops
     transpose
     unbind
     unsqueeze
+    unsafe_chunk
```

Comment:
The unsafe_* APIs should not show up in the docs. We don't want users using them directly.